test/test_subscription.py

```diff
@@ -321,3 +331,55 @@ def test_continuous_subscription(self):
                 self.retrieve(8889, fname, fileVersion=2, targetFile=tmp_path())
         finally:
             subscription_listener.close()
+
+    def test_https_subscription(self):
+        if os.environ.get("NGAS_TESTDB") is not None:
+            pytest.skip("Need two clusters for this test to run")
```

Comment:
I'd rather have a `@unittest.skipIf('NGAS_TESTDB' in os.environ, "Need two cluster for this test to run")` annotation rather than a pytest-specific way of skipping a test.

Reply:
I'll update that